frontend/src/App.js

```diff
@@ -91,14 +92,21 @@ class App extends Component {
             .then(json => this.parseResponse(json));
 
         // Persist the response into state of the App
-        this.setState({
-            hasSearched: true,
-            selectedFood: {
-                alias: results.contributors[0],
-                grams: results.grams[0]
-            },
-            results: results
-        });
+        if (Math.abs(results.totalImpact) > 1e-5) {
+            this.setState({
+                hasSearched: true,
+                selectedFood: {
+                    alias: results.contributors[0],
+                    grams: results.grams[0]
+                },
+                results: results
+            });
+        } else {
+            // Show a popper message in grocery list, suggesting the user add more food
+            this.setState({
+                showPopper: true
```

Comment:
- Bad communication: Love the idea!
- False positive: yeah agree it's a bit sketchy, so I reluctantly changed it :) Generally speaking though, I would like to take simplicity of code into the equation. The alternative here doesn't seem too complicated so I did it; otherwise we have been more hesitant in balancing the two objectives, especially since I don't think this would happen a lot.

Reply:
> False positive: yeah agree it's a bit sketchy, so I reluctantly changed it :) Generally speaking though, I would like to take simplicity of code into the equation. The alternative here doesn't seem too complicated so I did it; otherwise we have been more hesitant in balancing the two objectives, especially since I don't think this would happen a lot.

Hmm...keeping the code simple is fine. I agree we should take simplicity of code into the equation, and that (combined with frequency of occurrence) should affect priority of dealing with the issue. But the message that pops-up should at least be accurate/truthful, eg. 0 lbs of beef should have a message "Please add more food!" instead of "We don't have data on beef", so we should at least deal with this issue eventually. Best if we don't lie to people. 

Another thing is to consider that we aren't end-users so we can't assume real rate of occurrence of this happening out in the 'wild' (sometimes people don't see the usage of this tool the same as us, and maybe they'd run into this issue a lot).